modules/migration-mtc-release-notes-1-8-3.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-3_{context}"]
+= {mtc-full} 1.8.3 release notes
+
+[id=technical-changes-1-8-3_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.3 has the following technical changes:
+
+.Support added for {oadp-short} 1.3
+
+{mtc-short} 1.8.3 adds support to {oadp-first} as a dependency of {mtc-short} 1.8.z. link:https://issues.redhat.com/browse/MIG-1520[(MIG-1520)]
+
+[id="resolved-issues-1-8-3_{context}"]
+== Resolved issues
+
+This release has the following major resolved issues:
+
+.Restic backup does not work properly when the source workload is not quiesced
+
+The HTML file is not migrated when migrating an application with a route. The Restic backup does not function as expected when the source workload is unquiesced.
```

Comment:
It is not just the HTML file, this was used in the Steps to Reproduce:
1. Create a new project
2. Deploy an application with a route in it
3. Create a migration plan with the queisce option unchecked at the end
4. Run the migration
So the title `.Restic backup does not work properly when the source workload is not quiesced` does not match the description

Reply:
Changed text